R/wwinference.R

```diff
@@ -254,7 +268,7 @@ get_mcmc_options <- function(
 #'
 #'
 #' @param forecast_date a character string in ISO8601 format (YYYY-MM-DD)
-#' indicating the date that the forecast is to be made. Default is
+#' indicating the date that the forecast is to be made. Default is NULL
```

Comment:
What happens when the default is `NULL`? Will there be no forecast?

Reply:
Yeah so my goal here was to follow how some other R packages have a default of NULL when something is required. If the forecast date is missing, then the function errors with an informative error message telling the user to specify a forecast date. Almost everything else can follow defaults, but I think this should be required. 

See here: https://github.com/CDCgov/ww-inference-model/blob/48ac3a1c427154c6bd31f9dbabb3aed53ce6a849/R/wwinference.R#L312